templates/publisher/publicise.html

```diff
@@ -0,0 +1,516 @@
+              <img src="https://assets.ubuntu.com/v1/50df5ee8-JP-snap-store-white-uneditable.svg" />
+            </p>
+          </div>
+          <div class="col-3">
+            <a href="https://assets.ubuntu.com/v1/a1969218-JP.zip" class="p-button u-no-margin--bottom snapcraft-publicise__download">Download (zip)</a>
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-2">
+          <label>HTML:</label>
+        </div>
+        <div class="col-8">
+          <pre><code>&lt;a href="https://snapcraft.io/{{ snap_name }}" title="Get it from the Snap Store"&gt;
+  &lt;img src="https://assets.ubuntu.com/v1/dd5ab12a-JP-snap-store-black-uneditable.svg" alt="" /&gt;
+&lt;/a&gt;</code></pre>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-2">
+          <label>Markdown:</label>
+        </div>
+        <div class="col-8">
+          <pre><code>[![Get it from the Snap Store](https://assets.ubuntu.com/v1/dd5ab12a-JP-snap-store-black-uneditable.svg)](https://snapcraft.io/{{ snap_name }})</code></pre>
+        </div>
+      </div>
+    </div>
+
+    <div id="es_content" class="js-language-content u-hide">
+      <div class="row">
+        <div class="col-2">
+          <label>Snap store badge:</label>
+        </div>
+        <div class="col-8">
+          <div class="col-5">
+            <p class="snapcraft-publicise__images">
+              <img src="https://assets.ubuntu.com/v1/06b199ba-ES-snap-store-black-uneditable.svg" />
+              <img src="https://assets.ubuntu.com/v1/4ed2cce2-ES-snap-store-white-uneditable.svg" />
+            </p>
+          </div>
+          <div class="col-3">
+            <a href="https://assets.ubuntu.com/v1/b145cca1-ES.zip" class="p-button u-no-margin--bottom snapcraft-publicise__download">Download (zip)</a>
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-2">
+          <label>HTML:</label>
+        </div>
+        <div class="col-8">
+          <pre><code>&lt;a href="https://snapcraft.io/{{ snap_name }}" title="Instala desde Snap Store"&gt;
+  &lt;img src="https://assets.ubuntu.com/v1/06b199ba-ES-snap-store-black-uneditable.svg" alt="" /&gt;
+&lt;/a&gt;</code></pre>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-2">
+          <label>Markdown:</label>
+        </div>
+        <div class="col-8">
+          <pre><code>[![Instala desde Snap Store](https://assets.ubuntu.com/v1/06b199ba-ES-snap-store-black-uneditable.svg)](https://snapcraft.io/{{ snap_name }})</code></pre>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <p>The Snapcraft logo is licensed under CC BY-ND 2.0 UK, a registered trademark of Canonical Limited, 2018. Full license can be found on <a href="#" class="js-show-license">here</a>.</p>
+  </div>
+
+  <div id="license" class="u-hide">
```

Comment:
That's a bit weird way of showing the licence. Especially that clicking anything when it's open closes it. Could we have it on a separate page, like `/publicise/license` or something? That would just have this text of the licence on it and would be simply linked?